staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -68,15 +73,56 @@ var (
 		},
 		[]string{"result"},
 	)
+
+	authorizationAttemptsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "authorization_attempts_total",
+			Help:           "Counter of authorized attempts, result contains 'allowed', 'forbidden', 'no-opinion' and 'error'.",
```

Comment:
nit:
```suggestion
			Help:           "Counter of authorization attempts broken down by result. It can be either 'allowed', 'forbidden', 'no-opinion' or 'error'.",
```

Reply:
Thanks for the reminder and I have corrected